content/millennium/r4/individuals/related-person.md

```diff
@@ -0,0 +1,105 @@
+---
+title: RelatedPerson | R4 API
+---
+
+# RelatedPerson
+
+* TOC
+{:toc}
+
+## Overview
+
+The RelatedPerson resource provides information about a person who is involved in the care of a patient but has no formal responsibility. RelatedPersons typically have a personal relationship with the patient such as a spouse, relative, friend, guardian, or attorney. RelatedPersons are commonly a source of information about a patient.
+
+The distinction between Practitioner and RelatedPerson is a practitioner cares for multiple patients on behalf of a healthcare facility where a RelatedPerson performs care tasks for a specific patient and is not associated with any healthcare facility.
+
+* [RelatedPerson id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [RelatedPerson Identifier](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.identifier){:target="_blank"}
+* [Active](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.active){:target="_blank"} 
+* [Patient](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.patient){:target="_blank"}
+* [Relationship](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.relationship){:target="_blank"}
+* [Related person's name](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.name){:target="_blank"}
+* [Telecom information (may include phone and email)](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.telecom){:target="_blank"}
+* [Gender (administrative)](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.gender){:target="_blank"}
+* [Date of birth](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.birthDate){:target="_blank"}
+* [Address](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.address){:target="_blank"}
+* [Communication (preferred language)](https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.communication){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:related_person, :r4) %>
+
+## Search
+
+Search for RelatedPersons that meet supplied query parameters:
+
+    GET /RelatedPerson?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required?                                         | Type          | Description
+--------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------
+ `_id`        | This or `patient` or `identifier` | [`token`]     | The logical resource id associated with the resource.
+ `identifier` | This or `_id` or `patient`        | [`token`]     | A RelatedPerson identifier.
+ `patient`    | This or `_id` or `identifier`     | [`reference`] | A reference to a patient related which the RelatedPerson is associated. Example: `14067892`
```

Comment:
"A reference to a patient related which the RelatedPerson is associated." doesn't make sense to me

maybe one of:
"A reference to a patient to which the RelatedPerson is associated."
"A reference to a patient associated with the RelatedPerson."

Reply:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/a6cc265db68cabf4cb1d9abc1dec64f71d1c0b86#diff-b47a4cc256449f5aa96d433a01e1ae01R50

Reply:
need a period between RelatedPerson and Example

Reply:
Added period
https://github.com/cerner/fhir.cerner.com/pull/254/commits/a61676ce8e25adc46975ea912b5016edde8a8b6e#diff-b47a4cc256449f5aa96d433a01e1ae01R50